### ISSUE TITLE
`CellCoords` is not properly exported, while being a required type for some API methods.

### ISSUE THREAD

**Issue by jansiegel** [handsontable/handsontable#9755]
### Description
When trying to import `CellCoords`, we're getting an error of 

```
Module not found: Error: Package path ./3rdparty/walkontable/src/cell/coords is not exported
```

At the same time, the `CellCoords` object type is required as an argument in some public API methods, for example, https://handsontable.com/docs/api/hooks/#afteroncellcontextmenu.

### Steps to reproduce

1. Import `CellCoords` from `handsontable/3rdparty/walkontable/src/cell/coords`
2. See the error.

**Comment by AMBudnik** [comment]
Note to myself: Inform [forum](https://forum.handsontable.com/t/error-when-using-cellcoords-from-walkontable/6307/5)

**Comment by budnix** [comment]
I think that the `CellCoords` class should stay private (there should be no export). By private I mean the developer should not be able to create instances as there is no public API that requires passing such an object. At the same time, the developer can use that instances and their methods (as mentioned in the issue) to solve some problems [as documented](https://handsontable.com/docs/javascript-data-grid/api/cell-coords/).

**Comment by dagroe** [comment]
If I have a component which passes functions to handsontable hooks, e.g.

```typescript
import Handsontable from 'handsontable';
import CellCoords from 'handsontable/3rdparty/walkontable/src/cell/coords';

export interface Args extends Handsontable.GridSettings {
  settings?: Handsontable.GridSettings;
  onMouseEvent(
    event: MouseEvent,
    coords: CellCoords,
    element: HTMLTableCellElement
  ): void;
}

export default class Table extends Component<Args> {
  initHot(): Handsontable {
    const hotInstance = new Handsontable(this.hotElement, settings);
    // register hook for mouse events:
    if (this.args.onMouseEvent) {
      Handsontable.hooks.add(
        'afterOnCellMouseOver',
        this.args.onMouseEvent,
        hotInstance
      );
    }
    return hotInstance;
  }
}
```

I guess I need `CellCoords` to type the function `onMouseEvent` that gets passed to `afteroncellmouseover` hook, as documented here: https://handsontable.com/docs/javascript-data-grid/api/hooks/#afteroncellmouseover

**Comment by budnix** [comment]
Good point, @dagroe 👍 Just forgot about typings. After thinking about it, I change my mind. However, I'd propose to reexport the coords classes within the main package name, something like `import Handsontable, { CellCoords, CellRange } from 'handsontable'` instead of `'handsontable/3rdparty/walkontable/src/cell/coords'`.

**Comment by AMBudnik** [comment]
Hi @dagroe 

I have good news. We just released Handsontable v12.3.0 which fixes the issues.
